gef-ui/component-tests/includes/application-preview/facilities.component-test.js

```diff
@@ -0,0 +1,61 @@
+const { FACILITY_TYPE, FACILITY_STAGE } = require('@ukef/dtfs2-common');
+const pageRenderer = require('../../pageRenderer');
+
+const page = 'includes/application-preview/facilities.njk';
+const render = pageRenderer(page);
+
+const params = {
+  facilities: {
+    data: [
+      { heading: FACILITY_TYPE.CASH, facilityId: 'issuedCash', stage: FACILITY_STAGE.ISSUED },
+      { heading: FACILITY_TYPE.CASH, facilityId: 'unissuedCash', stage: FACILITY_STAGE.UNISSUED },
+      { heading: FACILITY_TYPE.CONTINGENT, facilityId: 'issuedContingent', stage: FACILITY_STAGE.ISSUED },
+      { heading: FACILITY_TYPE.CONTINGENT, facilityId: 'unissuedContingent', stage: FACILITY_STAGE.UNISSUED },
+    ],
+  },
+};
+
+describe(page, () => {
+  let wrapper;
+
+  describe("'Make a change' button", () => {
+    const makeAChangeButtonSelector = (facilityId) => `[data-cy="facility-${facilityId}-make-change-button"]`;
+
+    it('should not be rendered when showFacilityAmendmentButton is false', () => {
+      wrapper = render({
+        ...params,
+        canIssuedFacilitiesBeAmended: false,
+      });
+
+      wrapper.expectElement(makeAChangeButtonSelector('issuedCash')).notToExist();
+      wrapper.expectElement(makeAChangeButtonSelector('unissuedCash')).notToExist();
+      wrapper.expectElement(makeAChangeButtonSelector('issuedContingent')).notToExist();
+      wrapper.expectElement(makeAChangeButtonSelector('unissuedContingent')).notToExist();
```

Comment:
I found this a little bit confusing - the ids being `'issuedCash'` - would it be simpler to declare at the top
```
const issuedCashFacility = { heading: FACILITY_TYPE.CASH, facilityId: 'issuedCash', stage: FACILITY_STAGE.ISSUED };

const params = {
  facilities: {
    data: [
      issuedCashFacility,
    ],
  },
};

wrapper.expectElement(makeAChangeButtonSelector(issuedCashFacility.facilityId)).notToExist();
```

Reply:
Great idea, have changed thanks!

Reply:
Can we make the change @MarRobSoftwire proposes please, finding it confusing otherwise:

```
// nunjucks attribute
'data-cy': "facility-" + item.facilityId + "-make-change-button"

// component test
const makeAChangeButtonSelector = (facilityId) => `[data-cy="facility-${facilityId}-make-change-button"]`;
```

But we're not passing IDs in, strange mix of issued/unissued and the facility type?

```
makeAChangeButtonSelector('issuedCash')
```